### PR TITLE
docs: add Craig Spence to GDE list

### PR DESCRIPTION
**Pull request by phenomnomnominal**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -909,5 +909,13 @@
     "website": "https://medium.com/@abampakos",
     "bio": "Aristeidis is a front end web developer at Plex-Earth working with Angular. He is a co-organizer of the Angular Athens meetup and the author of Learning Angular - 3rd Edition. He is also a senior Angular instructor at Code.Hub where he teaches the Angular framework to other developers.",
     "groups": ["GDE"]
+  },
+  "phenomnominal": {
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:913]: Please change the key to be your GitHub handle :pray:

```suggestion
  "phenomnomnominal": {
```

**Review by petebacondarwin — APPROVED** [review]
Congrats @phenomnomnominal !

Reviewed-for: global-docs-approvers

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (petebacondarwin) [reply aio/content/marketing/contributors.json:913]: Oops good spot @gkalpak

Comment (phenomnomnominal) [reply aio/content/marketing/contributors.json]: Oops my bad! Thanks!

**Review by gkalpak — APPROVED** [review]
Thx, @phenomnomnominal! LGTM except you need to squash the commits together :grin:

<sub>
BTW, (for future PRs) you can keep follow-up changes in a separate commit (which is great for reviewing) and not have to squash commits together by using [fixup commits][1] :wink: 
</sub>

[1]: https://github.com/angular/angular/blob/1e151f9fe4a001b07b1e6aa906b1a83d1ced65a0/docs/FIXUP_COMMITS.md

**Comment by phenomnomnominal** [comment]
> Thx, @phenomnomnominal! LGTM except you need to squash the commits together 😁
> 
> BTW, (for future PRs) you can keep follow-up changes in a separate commit (which is great for reviewing) and not have to squash commits together by using [fixup commits][1] 😉

This is what I get for trying to be clever and use the Github UI 😅

**Comment by gkalpak** [comment]
💡 **Pro tip:** You can still have a fixup commit via the GitHub UI, if you manually enter the right commit message :wink:

**Pull request closed without merging**